client/components/tinymce/plugins/simple-payments/dialog/form.jsx

```diff
@@ -1,10 +1,14 @@
+/** @format */
```

Comment:
This is a Prettier marker that says that the file has been formatted with Prettier and will stay formatted -- there's a pre-commit hook for that. See @samouri's post at p4TIVU-6zb-p2.